bfvmm/src/vmcs/src/vmcs_intel_x64.cpp

```diff
@@ -33,8 +33,55 @@
 using namespace x64;
 using namespace intel_x64;
 
-vmcs_intel_x64::vmcs_intel_x64() :
-    m_vmcs_region_phys(0)
+void
+set_vm_control(uint32_t val, bool is_allowed0, bool is_allowed1,
+               uint64_t addr, const std::string &name, uint64_t mask)
+{
+    using namespace intel_x64::vmcs;
+
+    if (val == 0)
+    {
+        if (is_allowed0)
```

Comment:
Swap the order on this and drop the else. I'm actually surprised Clang Tidy didn't complain about that. But in general, if / else should be avoided as per the readability checks.